ui/v2.5/src/components/ScenePlayer/ScenePlayer.tsx

```diff
@@ -264,6 +266,17 @@ export const ScenePlayer: React.FC<IScenePlayerProps> = ({
 
   // Initialize VideoJS player
   useEffect(() => {
+    function isVrScene() {
+      if (!scene?.id || !vrTag) return false;
+
+      scene?.tags.map((tag) => {
+        if (vrTag == tag.name) {
+          return true;
+        }
+      });
+      return false;
```

Comment:
This currently returns false every time.

```suggestion
      return scene?.tags.some((tag) => {
        if (vrTag == tag.name) {
          return true;
        }
      });
```

Reply:
Ah, I see. I've gone ahead and pushed your suggested changes.

Reply:
You know I completely missed the suggested change you had here. I must have been multitasking when I looked at this that day.